roles/falcon_uninstall/tasks/uninstall.yml

```diff
@@ -13,7 +13,8 @@
   when:
     - ansible_system == "Linux"
 
-- block:
+- name: MacOS Block
```

Comment:
Please don't add these... this is intentional

Reply:
If you don't add it then you must ignore it via ansible-lint configuration. Then we would have to check with AAP folks if they are going to care about that or not (I think). Also, what's intentional about not naming a block? Just curious..